README.md

```diff
@@ -257,6 +257,10 @@ Setting up proper trust- and keystores has to be managed outside of the puppetdb
 
 If true, the module will attempt to connect to the database using the specified settings and fail if it is not able to do so. (defaults to true)
 
+####`database_path`
```

Comment:
I'd prefer this be called something more like `embedded_database_path` just to be explicit about the case it's used for, it will also signal to users that they need not care about it.